x-pack/plugins/uptime/server/lib/saved_objects/migrations.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { SavedObjectUnsanitizedDoc } from 'src/core/server';
+import { DynamicSettings } from '../../../common/runtime_types';
+
+export const add820Indices = (doc: SavedObjectUnsanitizedDoc<DynamicSettings>) => {
```

Comment:
nit: it's preferable to use the `SavedObjectMigrationFn<Input, Output>` type so that both the input and output has strong typing. In your function the output of your migration function isn't strictly typed so you loose some protection.

For this specific migration it probably doesn't matter too much, but generally speaking stricter typing helps catch uncaught exceptions that could cause downtime during an upgrade.